src/Writer.mo

```diff
@@ -108,6 +117,21 @@ module {
       # ";" # Nat8.toText(b); 
     };
 
+    func hexToRGB(hex: Text): ?(Nat8, Nat8, Nat8) {
+      let h = Text.trimStart(hex, #char '#');
+
+      switch (Hex.decode(h)){
+        case (#ok(bytes)){ 
+          if (bytes.size == 3){
```

Comment:
It looks like you have a small error here. Changing `bytes.size` to `bytes.size()` should fix it!